docs/core/preview3/migration/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: .NET Core Application Deployment | Microsoft Docs
+description: .NET Core project.json to csproj migration
+keywords: .NET, .NET Core, .NET Core migration
+author: blackdwarf
+ms.author: mairaw
+ms.date: 02/17/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: 1feadf3d-3cfc-41dd-abb5-a4fc303a7b53
+---
+
+# Migrating .NET Core projects
+
+[!INCLUDE[preview-warning](../../../includes/warning.md)]
+
+This document will cover migration scenarios for .NET Core projects. It will cover the following three migration scenarios:
+
+1. Migration from a valid latest schema of `project.json` to `csproj`
+2. Migration from DNX to a valid Preview 2 project.json
+3. Migration from RC3 and previous .NET Core csproj projects to the final format 
+
+## Migration from project.json to csproj
+Migration from project.json to csproj can be done via the [`dotnet migrate`](../tools/(dotnet-migrate.md) command-line tool or through Visual Studio 2017. Both of these migrations use the same underlying engine to migrate projects, which means that the results will be the same regardless of the method you choose. In high majority of cases, using one of these two ways to migrate the project.json to csproj is the only thing that is needed and no further manual editing of the csproj is necessary. 
+
+Visual Studio 2017 will migrate the project automatically on opening either the `xproj` file or the solution file which references `xproj` files. If a solution is opened, Visual Studio 2017 will automatically migrate all of the projects that are specified in the solution. 
```

Comment:
For this scenario, I think it'd be good to mention the One-time upgrade window that shows up and when you aceept that, then the projects are migrated automatically. And I'd show a screenshot of that.

on opening -> when you open

solution is opened -> solution file is opened

Do we need to mention anything about projects that didn't have a xproj or a solution earlier? Those need to use the command-line route